devel/clean

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+#
+# Cleans up after the build process by removing build artifacts and caches.
+#
+set -euo pipefail
+
+repo="$(cd "$(dirname "$0")/.."; pwd)"
+registry=nextstrain-local-registry
+build_dir="$repo"/build
+builder=nextstrain-builder
+
+
+echo "--> Deleting $registry container"
+
+if docker container inspect "$registry" &>/dev/null; then
+    "$repo"/devel/stop-localhost-registry
+else
+    echo "skipped; $registry container does not exist"
+fi
+
+
+echo "--> Deleting artifacts in $build_dir"
+
+if [[ -d "$build_dir" ]]; then
+    du -hs "$build_dir"
+    rm -rf "$build_dir"
+else
+    echo "skipped; $build_dir does not exist"
+fi
+
+
+echo "--> Deleting $builder (and its caches)"
+
+if docker buildx inspect "$builder" &>/dev/null; then
+    docker buildx du --builder "$builder"
+    docker buildx rm --builder "$builder"
+else
+    echo "skipped; $builder does not exist"
+fi
+
+
+echo "--> Deleting local images"
+
+for image in localhost:5000/nextstrain/base{,-builder}; do
```

Comment:
Fair enough. I skipped parameterization here because it seemed unnecessary, but I'll thread it thru.

(The overhead of threading thru params is why I prefer env vars, e.g. `LOCAL_REGISTRY=…`, over command-line arguments when there are many separate programs co-operating over shared resources.)

Reply:
I'd be ok with an env var for the local registry, if you want to make that change here.

Reply:
Parameterized with repush in 23a8431.

Reply:
I'll save any switch from command-line args to env vars for another time!